src/ui/components/MapBox/story/mapbox.js

```diff
@@ -0,0 +1,68 @@
+function storyMapBox(selectedStory) {
+
+	const token = "pk.eyJ1Ijoic2NvdHRpYWxlamFuZHJvIiwiYSI6ImNsNWJxNGo1YzAxOXUzZHE5b2k1OWxhZ3AifQ.O39Uy9OX7tjnNGJxAnEoiw";
+	// userPositionMark();
+	sendPointsToMap();
+
+	function userPositionMark() {
+		const config = {
+			access_token: token,
+			apis: [{
```

Comment:
Move apis inside configure{} function **m_props.apis**

Reply:
I thinking remove this options.